code/integration-tests/runtime-tests/test/tests/assets/rpcAssetsTests.ts

```diff
@@ -72,6 +80,74 @@ describe("[SHORT] rpc.assets Tests", function () {
     const KSM = result.find(e => hex_to_ascii(e.name.toString()) === "KSM")!;
     expect(PICA.id.toNumber()).to.equal(1);
     expect(KSM.id.toNumber()).to.equal(4);
+    expect(PICA.decimals.toNumber()).to.equal(12);
+
+    // Update KSM
```

Comment:
nit: Would prefer the `assetsRegistry.updateAsset` & `assetsRegistry.registerAsset` to be in a standalone test case. Instead of being part of the general `assets.listAssets` test.
This would keep those tests better maintainable and if they are failing in the future, easier to fix, or figure out what is wrong.

Reply:
Makes sense. I'll leave it for a later PR to get this merged.